src/AppInstallerCLICore/Workflows/SettingsFlow.cpp

```diff
@@ -14,14 +14,16 @@ namespace AppInstaller::CLI::Workflow
 
     namespace
     {
-        struct AdminSettings
+        struct ExportSettingsJson
         {
-            AdminSettings()
+            ExportSettingsJson()
             {
+                root["$schema"] = "https://aka.ms/winget-settings-export.schema.json";
                 root["adminSettings"] = Json::ValueType::objectValue;
+                root["userSettingsFile"] = Runtime::GetPathTo(Runtime::PathName::UserSettingsFileLocation).u8string();

```

Comment:
And this one should be the full path without the environment variable (thus not named `ForDisplay`).